src/Binance/CoinMFutures/API/ContinuousCandle.jl

```diff
@@ -0,0 +1,124 @@
+module ContinuousCandle
+
+export ContinuousCandleQuery,
+    ContinuousCandleData,
+    continuous_candle
+
+using Serde
+using Dates, NanoDates, TimeZones
+
+using CryptoAPIs.Binance
+using CryptoAPIs: Maybe, APIsRequest
+
+@enum TimeInterval m1 m3 m5 m15 m30 h1 h2 h4 h6 h8 h12 d1 d3 w1 M1
+
+@enum TypeOfContract PERPETUAL CURRENT_QUARTER NEXT_QUARTER
+
+Base.@kwdef struct ContinuousCandleQuery <: BinancePublicQuery
+    pair::String
+    contractType::TypeOfContract
+    interval::TimeInterval
+    limit::Maybe{Int64} = nothing
+    endTime::Maybe{DateTime} = nothing
+    startTime::Maybe{DateTime} = nothing
+end
+
+function Serde.ser_type(::Type{<:ContinuousCandleQuery}, x::TimeInterval)::String
+    x == m1  && return "1m"
+    x == m3  && return "3m"
+    x == m5  && return "5m"
+    x == m15 && return "15m"
+    x == m30 && return "30m"
+    x == h1  && return "1h"
+    x == h2  && return "2h"
+    x == h4  && return "4h"
+    x == h6  && return "6h"
+    x == h8  && return "8h"
+    x == h12 && return "12h"
+    x == d1  && return "1d"
+    x == d3  && return "3d"
+    x == w1  && return "1w"
+    x == M1  && return "1M"
+end
+
+function Serde.ser_value(::Type{<:ContinuousCandleQuery}, ::Val{:contractType}, v::String)
+end
```

Comment:
```suggestion
```

Reply:
I agree with @maxfadson and think that this is unnecessary here.